libeos-payg/util.h

```diff
@@ -16,13 +16,21 @@
  * LGPL. If you do not delete the provisions above, a recipient may use your
  * version of this file under the terms of either the MPL or the LGPL.
  */
-
```

Comment:
I suppose this stands out as an unrelated (unintentional?) formatting change.

I'm not really bothered by it.

Reply:
Fixed this